### PR TITLE
AC-682: Create phpcs static check for RestrictedCodeTest

### PR DESCRIPTION
**Pull request by svera**
This sniff checks files for usages of restricted classes in there.

### CODE REVIEW

File: Magento2/ruleset.xml [hunk 1]
```diff
@@ -142,6 +142,11 @@
         <type>error</type>
         <exclude-pattern>*\.xml$</exclude-pattern>
     </rule>
+    <rule ref="Magento2.Legacy.RestrictedCode">
+        <severity>10</severity>
+        <type>error</type>
+        <exclude-pattern>*\.xml$</exclude-pattern>
```
Comment (jcuerdo) [inline Magento2/ruleset.xml:148]: Why only exclude xml?

**Review by jcuerdo — APPROVED** [review]

File: Magento2/ruleset.xml (hunk 1 above)
Comment (svera) [reply Magento2/ruleset.xml:148]: @jcuerdo excluded also JS files

**Comment by svera** [comment]
@magento import pr to magento-commerce/magento-coding-standard

**Comment by m2-github-services** [comment]
@svera the Pull Request is successfully imported.

**Pull request merged**